mkt/constants/base.py

```diff
@@ -92,6 +93,18 @@
     'unlisted': STATUS_UNLISTED,
 }
 
+STATUS_FILE_CHOICES_API_v2 = {
```

Comment:
Could we limit this to the valid choices we know are possible for add-ons (and apps)?